jss-old-data/pom.xml

```diff
@@ -38,7 +38,7 @@
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.8</version>
+            <version>5.1.48</version>
```

Comment:
Any specific advantage or reason for this version? Why not say 8.0? Also, we don't use this module (jss-old-data anymore).

Reply:
The mysql version `5.1.8` has few vulnerabilities which are resolved in 5.1.41. So I upgraded it to latest patch version available. 

Why not say 8.0? Since upgrading to latest patch version itself resolves the issue and I am not sure about the impact of upgrading major versions